public/projects-list/scripts/glitch.js

```diff
@@ -4,10 +4,101 @@ var $ = require("jquery");
 
 module.exports = {
   init: function() {
-    let banner = $(".glitch-banner");
-    let cta = $(".glitch-cta.underlay");
-    if (banner && cta) {
+    let banner = $(".glitch-banner"),
+      cta = $(".glitch-cta.underlay"),
+      content = $(".content", cta),
+      close = $(".cta-close", content),
```

Comment:
you probably need to add a listener for this too so that the text for the buttons resets.

STR:
1. Click on "export" for a project.
2. Click on "Start exporting", and watch it fail.
3. Close the dialog.
4. Reopen the dialog and it will still show the button saying "Exporting..."

Reply:
Hm, https://github.com/mozilla/thimble.mozilla.org/pull/2673/files/f18a7c98d140ab867696e5ffcc9d54ed8d733a9a#diff-188a9152d1759a40d1395fa702a49491R77 should already do that, I'll have a look at why it's not.